docs/src/testing/testing.md

```diff
@@ -72,3 +72,25 @@ Failure data:
 
 Tests: 0 passed, 1 failed, 0 skipped
 ```
+
+## Expected failures
+
+Sometimes you want to mark a test as expected to fail. This is useful when you are working on a feature that is not yet, or when you want to verify that an action fails as expected.
```

Comment:
```suggestion
Sometimes you want to mark a test as expected to fail. This is useful when you want to verify that an action fails as expected.
```
I belive if working on not yet implemented feature one should use `#[skip]` (not yet available in snforge) instead